packages/hooks/src/__tests__/useTotalAccountBalance.spec.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { mockStore, StoreProvider } from '@deriv/stores';
+import { renderHook } from '@testing-library/react-hooks';
+import useTotalAccountBalance from '../useTotalAccountBalance';
```

Comment:
@shayan-deriv There are some TS errors in the hook itself, Would you help to resolve them 🙏🏻 

<img width="1393" alt="image" src="https://user-images.githubusercontent.com/72082844/236720723-2f05271b-c263-4fdc-bc6a-e3af581f7000.png">